Marlin/src/lcd/e3v2/proui/dwin.cpp

```diff
@@ -1633,18 +1619,16 @@ void dwinLevelingDone() {
 
 #if HAS_PID_HEATING
 
-  void dwinStartM303(const bool seenC, const int c, const bool seenS, const heater_id_t hid, const celsius_t temp) {
-    if (seenC) hmiData.pidCycles = c;
-    if (seenS) {
-      switch (hid) {
-        #if ENABLED(PIDTEMP)
-          case 0 ... HOTENDS - 1: hmiData.hotendPidT = temp; break;
-        #endif
-        #if ENABLED(PIDTEMPBED)
-          case H_BED: hmiData.bedPidT = temp; break;
-        #endif
-        default: break;
-      }
+  void dwinStartM303(const int count, const heater_id_t hid, const celsius_t temp) {
+    hmiData.pidCycles = c;
+    switch (hid) {
```

Comment:
`hmiData.pidCycles = c;`
should that be `count` instead of `c`?

Reply:
More descriptive names are better for generic function definitions. It's not uncommon for short-lived G-code internal variables to use single letters to hint they pertain to a parameter letter. Each naming style fits in context and they don't necessarily need to influence each other.